service/metrics/config.go

```diff
@@ -15,6 +15,31 @@ type Config struct {
 	Collect map[string]Collector
 }
 
+type NamedCollector struct {
```

Comment:
need json aliases here since it will be passed from PHP

Reply:
Done